src/current/_includes/releases/v22.2/v22.2.15.md

```diff
@@ -0,0 +1,75 @@
+- Fixed a bug that prevented table statistics from loading on the [Databases page Tables view](https://www.cockroachlabs.com/docs/v22.2/ui-databases-page#tables-view). [#109921][#109921]
+- Fixed a bug that could cause a transaction performing multiple parallel [foreign key](https://www.cockroachlabs.com/docs/v22.2/foreign-key) checks to return a `concurrent txn use detected` error. [#109850][#109850]
+- Fixed a bug where dependencies on [sequences](https://www.cockroachlabs.com/docs/v22.2/create-sequence) from tables would be reported with the wrong value for the `classid` column in the `pg_catalog.pg_depend` table. [#110211][#110211]
+- Fixed edge cases in [decimal](https://www.cockroachlabs.com/docs/v22.2/decimal) and [float](https://www.cockroachlabs.com/docs/v22.2/float) evaluation for division operators. `'NaN'::DECIMAL / 0` will now return `NaN` instead of a division-by-zero error, and `0 / 'inf'::DECIMAL` will return `0` instead of `0E-2019`. [#110298][#110298]
+- The `cockroach debug pebble` command now works correctly with [encrypted stores](https://www.cockroachlabs.com/docs/v22.2/security-reference/encryption#cockroachdb-self-hosted-clusters) which do not use the default `cockroach-data` path without having to also pass `--store`. [#110509][#110509]
+- When upgrading from previous releases, it was possible that ambiguous references to [sequences](https://www.cockroachlabs.com/docs/v22.2/create-sequence) would fail to be resolved by ID if they were in the same schema. Additionally, the logging is improved for ambiguous cases where the reference cannot be resolved. [#110721][#110721]
+- Fixed a bug where [`RESET `](https://www.cockroachlabs.com/docs/v22.2/row-level-ttl#reset-a-storage-parameter-to-its-default-value)[(`ttl_expire_after`)](https://www.cockroachlabs.com/docs/v22.2/row-level-ttl#using-ttl_expire_after) could incorrectly remove [`ttl_expiration_expression`](https://www.cockroachlabs.com/docs/v22.2/row-level-ttl#using-ttl_expiration_expression). [#110750][#110750]
+- Fixed a bug where [`format_type`](https://www.cockroachlabs.com/docs/v22.2/functions-and-operators#compatibility-functions) builtin did not honor `typemod` information for [array](https://www.cockroachlabs.com/docs/v22.2/array) types, leading to incorrect output. [#110939][#110939]
+- External connection URLs now accept the scheme `azure-blob` for connections to [Azure](https://www.cockroachlabs.com/docs/v22.2/use-cloud-storage#url-format) Blob Storage and the scheme `azure-kms` for connections to Azure KMS. For backward compatibility, schemes `azure` and `azure-storage` schemes continue to work for connections to Azure Blob Storage.
```

Comment:
There is no PR link for this one

also not finding anything with [this search](https://github.com/cockroachdb/cockroach/pulls?q=%22for+connections+to+Azure+KMS%22)